lib/paperclip/storage/google_drive.rb

```diff
@@ -95,6 +95,24 @@ def flush_deletes
         end
         @queued_for_delete = []
       end
+
+      def copy_to_local_file(style, local_dest_path)
+        client = google_api_client
+        drive = client.discovered_api('drive', 'v2')
+        searched_id = search_for_title(path(style))
+        metadata =metadata_by_id(searched_id)
+        uri = URI.parse(metadata['downloadUrl'])
+        Net::HTTP.start(uri.host, uri.port, use_ssl: true) do |http|
+          File.open(local_dest_path, "wb") do |file|
+            http.request_get(uri.request_uri, 'Authorization' => "Bearer #{client.authorization.access_token}") do |resp|
+              resp.read_body do |segment|
+                file.write(segment)
+              end
+            end
+          end
+        end
+        true
```

Comment:
I can't recall the specifics now, probably to match the official Paperclip API, or to obscure the HTTP Response object.